pkg/actors/requests_responses.go

```diff
@@ -76,6 +78,24 @@ type ReminderResponse struct {
 	Period  string `json:"period"`
 }
 
+// MarshalJSON is a custom JSON marshaler that encodes the data as JSON.
+func (r *ReminderResponse) MarshalJSON() ([]byte, error) {
+	type responseAlias ReminderResponse
+	m := struct {
+		Data []byte `json:"data,omitempty"`
+		*responseAlias
+	}{
+		responseAlias: (*responseAlias)(r),
+	}
+
+	var err error
+	m.Data, err = json.Marshal(r.Data)
+	if err != nil {
+		return nil, err
+	}
+	return json.Marshal(m)
```

Comment:
why is a double JSON marshal needed?

Reply:
Explained below and in the comments on the code.